ui/src/assets/common.scss

```diff
@@ -21,12 +21,16 @@
   --topbar-height: 44px;
   --monospace-font: "Roboto Mono", monospace;
   --track-shell-width: 250px;
-  --track-border-color: #00000025;
   --anim-easing: cubic-bezier(0.4, 0, 0.2, 1);
   --selection-stroke-color: #00344596;
   --selection-fill-color: #8398e64d;
   --overview-timeline-non-visible-color: #c8c8c8cc;
   --details-content-height: 280px;
+  --main-background-color:#ffffff;
+  --main-foreground-color:#222;
+  --bg-alt-color:#808080;
```

Comment:
Should this be named `--alt-background-color` to align with `--main-background-color`? Otherwise it looks like it belongs to a different naming scheme and therefore is somehow used entirely differently.